doc/docs.md

```diff
@@ -30,7 +30,7 @@ You need `git`, and a C compiler like `gcc` or `msvc`:
 ```bash
 git clone https://github.com/vlang/v
 cd v
-make
+make.bat -gcc
```

Comment:
Good spot.
`make.bat -tcc` will be better for most people imho.

Reply:
The reason is that we have a prebuilt tcc that the .bat file will simply download, and it is smaller than the gcc binary packages from say http://winlibs.com/

Reply:
Or just `make.bat` - I think in this case `make.bat` will by default use tcc also.